composer.json

```diff
@@ -17,7 +17,7 @@
     "require": {
         "php": "^7.2",
         "psr/http-message": "^1.0",
-        "willdurand/negotiation": "^2.0"
+        "willdurand/negotiation": "^3.0"
```

Comment:
I would make this `"willdurand/negotiation": "^2.0 | ^3.0"` as it should still be compatible with 2.x if there are no breaking changes

Reply:
Actually this is fine it was only a naming change and they actually recommend upgrading to 3.0 for php 8 compat: https://github.com/willdurand/Negotiation/releases/tag/3.0.0